managers/views.py

```diff
@@ -44,3 +45,32 @@ class ProjectDetail(APIView):
     def get(self, request, pk):
         project = get_object_or_404(Project, pk=pk)
         return Response({'project': project})
+
+
+class ProjectUpdate(APIView):
```

Comment:
Same here, lets discuss how this should actually work - as API, or as a classic django view with template.

Reply:
It will be done in separate pull request

Reply:
:+1: done